backend/controllers/appointments/prepostAppoinments.js

```diff
@@ -6,6 +6,7 @@ const {
 
 const { serviceUnavailableMessage } = require('../../common-messages')
 const { toAppointmentDetailsSummary } = require('./appointmentsService')
+const { notifyClient } = require('../../shared/notifyClient')
```

Comment:
Would be nice to see notifyClient as an injectable dependency like the others `({ elite2Api, oauthApi, appointmentsService, existingEventsService, logError }) `

Reply:
I was wondering between that and importing it where needed. It does mean we're initialising it multiple times this way, which I've just about come around to thinking about :D